docs/install-mlrun-ce/mlrun-ce-installation-notes.md

```diff
@@ -0,0 +1,49 @@
+# MLRun CE installation notes
+
+This page lists additional steps or configuration options you may need to follow for non-default MLRun CE installations.
+
+**In this section**
+
+- [Advanced chart configuration](#advanced-chart-configuration)
+- [Opt out of components](#opt-out-of-components)
+- [Using NFS storage](#using-nfs-storage)
+- [Configuring the online feature store](#configuring-the-online-feature-store)
+- [Installing Spark Operator on non-mlrun namespace](#installing-spark-operator-on-non-mlrun-namespace)
+
+## Advanced chart configuration
+
+Configurable values are documented in the `values.yaml`, and the `values.yaml` of all sub charts. Override those [in the normal methods](https://helm.sh/docs/chart_template_guide/values_files/).
```

Comment:
It might be useful to point to the actual default `values.yaml` file in GH, so users can reference. For example, when opting out of components - this will allow users to see what components are there (other than the examples we give).